manifests/backup/backup-s3-br.yaml

```diff
@@ -4,20 +4,27 @@ kind: Backup
 metadata:
   name: demo1-backup-s3
   namespace: test1
+  # annotations:
+    # iam.amazonaws.com/role: "arn:aws:iam::123456789:role"
 spec:
-  #backupType: full
+  # backupType: full
+  # serviceAccount: myServiceAccount
   br:
-    pd: 10.233.40.168:2379
-    # ca: <ca>
-    # cert: <cert>
-    # key: <key>
+    cluster: myCluster
```

Comment:
Is this changed from pd address to clusterName? @DanielZhangQD

Reply:
yes, I modify this field when I add tls to br

Reply:
Yes, it's changed in #1836.